content/sensu-go/5.0/api/overview.md

```diff
@@ -35,9 +35,56 @@ Sensu makes no guarantee that an alpha or beta API will be maintained for any pe
 Alpha versions should be considered under active development and may not be published for every release.
 Beta APIs, while more stable than alpha versions, offer similarly short-lived lifespans and also provide no guarantee of programmatic conversions when the API is updated.
 
-### Access control
+## Access control
 
-The Sensu API requires authentication using a JWT access token.
+The Sensu API requires authentication using a JWT token.
+
+## The `/auth` API endpoint
+The `/auth` endpoint lets you generate API tokens using your Sensu username and password.
+
+### Example {#auth-example}
+
+The following example requests access credentials using the default Sensu admin user (username `admin`, password `P@ssw0rd!`).
+
+{{< highlight shell >}}
+curl -u 'admin:P@ssw0rd!' -s http://localhost:8080/auth
+{{< /highlight >}}
+
+The output provides an access token and a refresh token.
+
+{{< highlight shell >}}
+{
+  "access_token": "eyJhbGciOiJIUzI1NiIs...",
+  "expires_at": 1544582187,
+  "refresh_token": "eyJhbGciOiJIUzI1NiIs..."
+}
+{{< /highlight >}}
+
+You can use the refresh token as a long-lived token or use the access token as a short-lived token.
```

Comment:
While it might work, it's actually a bug; refresh tokens should not be used as access tokens. It could cause RBAC issues if a user belongs to one or more groups, since the refresh token does not contain a `group` field in its claims. I opened https://github.com/sensu/sensu-go/issues/2534 to discuss this issue and the possibility to create API keys instead!